### PR TITLE
removes unused variables to fix lint errors

### PR DESCRIPTION
**Pull request by phishy**
## What does this PR do?

removes unused variables to fix lint errors inside `@calcom/web` except `playwright`

https://github.com/calcom/cal.com/actions/runs/3831751619/jobs/6521254368#step:5:439

**Environment**: Staging(main branch)

## Type of change

- [x] Chore (refactoring code, technical debt, workflow improvements)

## How should this be tested?

- [ ] ensure the tests pass

### CODE REVIEW

File: apps/web/pages/apps/[slug]/index.tsx [hunk 1]
```diff
@@ -20,7 +20,7 @@ const components = {
       <a {...otherProps} />
     </Link>
   ),
-  img: ({ src = "", alt = "", placeholder, ...rest }: JSX.IntrinsicElements["img"]) => (
+  img: ({ src = "", alt = "", ...rest }: JSX.IntrinsicElements["img"]) => (
```
Comment (zomars) [inline apps/web/pages/apps/[slug]/index.tsx:23]: I think the placeholder is being kept out of the `rest` on purpose.

**Review by zomars — CHANGES_REQUESTED** [review]
Thank you for your contribution 🙏 let fix the checks first tho.

**Comment by phishy** [comment]
The checks have so much noise in them they're unreadable.

**Comment by zomars** [comment]
Care to elaborate @phishy ?

**Comment by phishy** [comment]
@zomars Sure. Let's play a game called find the error in this pile of words. https://github.com/calcom/cal.com/actions/runs/3849187325/jobs/6557886188

**Comment by phishy** [comment]
Why use Typescript and linters, if nobody will adhere to them?

**Comment by zomars** [comment]
> @zomars Sure. Let's play a game called find the error in this pile of words. https://github.com/calcom/cal.com/actions/runs/3849187325/jobs/6557886188

- ctrl + f
- type " error "

image

Any suggestions to improve the error finding experience are always welcome 🙏🏽 

May you be well

**Comment by phishy** [comment]
My suggestion is to remove all the warning, such that the errors are easier to see.

**Comment by PeerRich** [comment]
why was this closed? looked like good work no?

**Pull request closed without merging**